packages/teleport-generator-core/src/validator/index.ts

```diff
@@ -62,14 +62,14 @@ export default class Validator {
   }
 
   public validateProjectContent(input: ProjectUIDL): ValidationResult {
-    const verifyRoutes = checkRouteDefinition(input)
+    const verifyRoutes = utils.checkRouteDefinition(input)
     let errors = verifyRoutes.errors
 
     if (verifyRoutes.errors.length === 0) {
-      const verifyRootComponent = checkRootComponent(input)
+      const verifyRootComponent = utils.checkRootComponent(input)
 
-      const verifyComponentNaming = checkComponentNaming(input)
-      const verifyComponentExistence = checkComponentExistence(input)
+      const verifyComponentNaming = utils.checkComponentNaming(input)
```

Comment:
is there any benefit from having these functions return an array of errors inside an object? can't the functions return just the array? so when it's empty, nothing is displayed from that area